src/crypto/src/db3_document.rs

```diff
@@ -0,0 +1,267 @@
+use crate::db3_address::DB3Address;
+use crate::id::{AccountId, DocumentId, TxId};
+use bson::spec::BinarySubtype;
+use bson::Document;
+use bson::{Binary, Bson, RawDocumentBuf};
+use db3_error::{DB3Error, Result};
+use fastcrypto::traits::ToFromBytes;
+use serde_json::{Map, Value};
+
+#[derive(Debug)]
+pub struct DB3Document {
```

Comment:
moving  the DB3Document to src/storage/src

Reply:
Therefore, we will include db3-storage in the cmd/cargo.tmol

Reply:
Let's keep it in the crypto and will fix it in the next PR.